open_webui_plugins_builder/tests/test_config.py

```
@@ -0,0 +1,313 @@
+        ],
+    }
+    with pytest.raises(BuildError, match="must be a relative path"):
+        parse_config(raw, repo_root=tmp_path)
+
+
+def test_rejects_absolute_output_path(tmp_path: Path) -> None:
+    raw = {
+        "meta": {"schema_version": 1},
+        "settings": {"local_import_roots": ["owui_ext.shared"]},
+        "targets": [
+            {"name": "invalid", "source": "src/a.py", "output": "/tmp/out.py"}
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/out.py"

(S108)